Building fox requires FCFLAGS and a cleared LIBS.
This is fixed in QM-E 6.3

Miguel Dias Costa, 20190509
diff -ru qe-6.2.1.orig/install/extlibs_makefile qe-6.2.1/install/extlibs_makefile
--- qe-6.2.1.orig/install/extlibs_makefile	2017-12-11 17:52:53.000000000 +0100
+++ qe-6.2.1/install/extlibs_makefile	2019-05-09 08:04:29.548203090 +0200
@@ -80,7 +80,8 @@
 	if test ! -d ../FoX; then \
         mkdir ../FoX; \
 	(gzip -dc ../archive/fox.tgz | (cd ../FoX; tar -xvf -)); \
-	cd ../FoX/fox/; export FC=$(F90); ./configure --prefix=$(TOPDIR)/FoX ;\
+	cd ../FoX/fox/; export FC=$(F90); export FCFLAGS="$(FOX_FLAGS)"; \
+	unset LIBS; ./configure --prefix=$(TOPDIR)/FoX ;\
 	$(MAKE) install; cd ../; rm -fr fox;fi
 # ELPA
 libelpa : libelpa_$(ELPA_LIBS_SWITCH)
